### PR TITLE
pr for ccd definition

### PR DESCRIPTION
**Pull request by velimd**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/integrationTest/resources/application.properties [hunk 1]
```diff
@@ -44,7 +44,7 @@ case.maintenance.amend-petition-draft-refusal.context-path=/casemaintenance/vers
 ###############################################
 #  Core Case Data Variables                   #
 ###############################################
-core_case_data.api.url=${CASE_DATA_STORE_BASEURL:http://ccd-data-store-api-aat.service.core-compute-aat.internal}
+core_case_data.api.url=http://ccd-data-store-api-nfdiv-ccd-definitions-pr-16.service.core-compute-preview.internal
```
Comment (nitinprabhu) [inline src/integrationTest/resources/application.properties:47]: Do you really need to change this ? in preview template you have already set `CASE_DATA_STORE_BASEURL` so what's existing should work ?

**Comment by nitinprabhu** [comment]
> # Description
> Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
> 
> Fixes # (issue)
> 
> ## Type of change
> Please delete options that are not relevant.
> 
> * [ ]  Bug fix (non-breaking change which fixes an issue)
> * [ ]  New feature (non-breaking change which adds functionality)
> * [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
> * [ ]  This change requires a documentation update
> 
> # How Has This Been Tested?
> Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
> 
> **Test Configuration**:
> 
> * Hardware:
> * O/S and version:
> * JDK:
> 
> # Checklist:
> * [ ]  My code follows the style guidelines of this project
> * [ ]  I have performed a self-review of my own code
> * [ ]  I have commented my code, particularly in hard-to-understand areas
> * [ ]  I have made corresponding changes to the documentation
> * [ ]  My changes generate no new warnings
> * [ ]  I have added tests that prove my fix is effective or that my feature works
> * [ ]  New and existing unit tests pass locally with my changes
> * [ ]  Any dependent changes have been merged and published in downstream modules

Would be nice to remove the default template and only include description of changes

File: src/integrationTest/resources/application.properties (hunk 1 above)
Comment (velimd) [reply src/integrationTest/resources/application.properties:47]: True. im not sure what the colon in template literals is for tbh, i assumed its a condition if its null use other one but im not sure. This branch isnt going to be used so i think think it matters much.

Comment (nitinprabhu) [reply src/integrationTest/resources/application.properties:47]: yes it is a condition if first is not set use the other one by default. ok if it is not going to be used it should be fine thanks

**Comment by velimd** [comment]
> Would be nice to remove the default template and only include description of changes

im actually suprised at how long this one is to the normal ones i see on hmcts prs

**Pull request closed without merging**